plotly/offline/offline.py

```diff
@@ -298,3 +309,128 @@ def plot(figure_or_data,
             ])
         else:
             return plot_html
+
+
+def plot_mpl(mpl_fig, resize=False, strip_style=False,
+             verbose=False, **kwargs):
+    '''
+    Convert a matplotlib figure to a plotly graph locally as an HTML document
+    or string
+
+    For more information on converting matplotlib visualizations to plotly
+    graphs, call help(plotly.tools.mpl_to_plotly)
+
+    For more information on creating plotly charts locally as an HTML document
+    or string, call help(plotly.offline.plot)
+
+    :param (matplotlib figure) mpl_fig: matplotlib figure to convert to a
+        plotly graph
```

Comment:
:cow2: super picky, but I think we should use `Plotly` when we talk about a Plotly graph and `plotly` when we talk about this module. Does that seem right? This is obviously not blocking.